test/e2e/specs/publishing.test.js

```diff
@@ -7,22 +7,45 @@ import {
 } from '../support/utils';
 
 describe( 'Publishing', () => {
-	beforeEach( async () => {
-		await newPost();
+	describe( 'a post', () => {
```

Comment:
FWIW we could avoid code duplication by iterating types if all other logic is basically the same:

```js
[ 'post', 'page' ].forEach( ( postType ) => {
	describe( postType, () => {
		beforeEach( async () => {
			await newPost( postType );
		} );

		// ...
	} );
} );
```